packages/datagrid/src/datagrid.ts

```diff
@@ -4085,33 +4119,17 @@ export class DataGrid extends Widget {
     for (let i = 0; i < rowColumnCount; i++) {
       const numCols = dataModel.rowCount('column-header');
       /*
-        Calculate the maximum text width vertically, across
+        Calculate the maximum text width, across
         all nested columns under a given row index.
       */
       let maxWidth = 0;
       for (let j = 0; j < numCols; j++) {
-        const cellValue = dataModel.data('corner-header', j, i);
-
-        // Basic CellConfig object to get the renderer for that cell.
-        let config = {
-          x: 0,
-          y: 0,
-          width: 0,
-          height: 0,
-          region: 'column-header' as DataModel.CellRegion,
-          row: 0,
-          column: i,
-          value: null as any,
-          metadata: DataModel.emptyMetadata
-        };
-
-        // Get the renderer for the given cell.
-        const renderer = this.cellRenderers.get(config) as TextRenderer;
-
-        // Use the canvas context to measure the cell's text width
-        const gc = this.canvasGC;
-        gc.font = CellRenderer.resolveOption(renderer.font, config);
-        const textWidth = gc.measureText(cellValue).width;
+        const textWidth = this._getCellTextWidth(
```

Comment:
NB that this does not translate to the same code as the original.

Initially, the `cellValue` was procured from the `corner-header`, however, the `config` would refer to the `column-header` area. I guess that's a bug?